src/lib/select/select.spec.ts

```diff
@@ -1800,6 +1800,31 @@ describe('MdSelect', () => {
         expect(options[2].getAttribute('aria-selected')).toEqual('false');
       });
 
+      it('should show ripples for options by default', () => {
+        expect(overlayContainerElement.querySelectorAll('.mat-ripple-element').length)
+          .toBe(0, 'Expected no ripples to show up initially.');
+
+        dispatchFakeEvent(options[0], 'mousedown');
+        dispatchFakeEvent(options[0], 'mouseup');
+
+        expect(overlayContainerElement.querySelectorAll('.mat-ripple-element').length)
+          .toBe(1, 'Expected one ripple to show up after a fake click.');
+      });
+
+      it('should not show ripples for options if disableRipple is set to true', () => {
+        fixture.componentInstance.disableRipple = true;
+        fixture.detectChanges();
+
+        expect(overlayContainerElement.querySelectorAll('.mat-ripple-element').length)
+          .toBe(0, 'Expected no ripples to show up initially.');
+
+        dispatchFakeEvent(options[0], 'mousedown');
```

Comment:
Not sure about that. This test would just ensure that the `disableRipple` property changed properly and also got reflected properly in the view. Verifying for example that `markForCheck` is placed properly.

I agree on adding a test for the property itself though.